repo/packages/B/beta-mesosphere-jupyter-service/0/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.11",
+  "name": "beta-mesosphere-jupyter-service",
+  "version": "1.3.0-0.35.4-beta",
+  "scm": "https://github.com/dcos-labs/dcos-jupyterlab-service",
+  "maintainer": "support+mesosphere-jupyter-service@mesosphere.com",
```

Comment:
`maintainer` for the current jupyterlab package is `https://dcos.io/community`. Is it intended to add Mesosphere support from this release?

Reply:
For this particular service, yes, since this will be a pass-through to me and the Data Science Engineering Team so we can incorporate feedback into what will eventually be the GA product.